.github/workflows/coverage-comment.yml

```
@@ -0,0 +1,239 @@
+        image: postgres:15
+        env:
+          POSTGRES_USER: postgres
+          POSTGRES_PASSWORD: postgres
+          POSTGRES_DB: eve_tracker_test
+        options: >-
+          --health-cmd pg_isready
+          --health-interval 10s
+          --health-timeout 5s
+          --health-retries 5
+        ports:
+          - 5432:5432
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)